<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Sudoku Solver</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QTableWidget" name="sudokuTable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="cornerButtonEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>9</number>
       </property>
       <property name="columnCount">
        <number>9</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>40</number>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <item row="0" column="0">
        <property name="text">
         <string/>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="1">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="2">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="3">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="4">
        <property name="text">
         <string/>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="5">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="6">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="7">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="0" column="8">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string/>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="1" column="1">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="1" column="2">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="1" column="3">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="1" column="4">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="1" column="5">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="1" column="6">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="1" column="7">
        <property name="text">
         <string/>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="1" column="8">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="2" column="0">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="2" column="1">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="2" column="2">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="2" column="3">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="2" column="4">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="2" column="5">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="2" column="6">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="2" column="7">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="2" column="8">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="3" column="0">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="3" column="1">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="3" column="2">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="3" column="3">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="3" column="4">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="3" column="5">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="3" column="6">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="3" column="7">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="3" column="8">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="4" column="0">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="4" column="1">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="4" column="2">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="4" column="3">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="4" column="4">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="4" column="5">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="4" column="6">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="4" column="7">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="4" column="8">
        <property name="text">
         <string/>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="5" column="0">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="5" column="1">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="5" column="2">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="5" column="3">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="5" column="4">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="5" column="5">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="5" column="6">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="5" column="7">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="5" column="8">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="6" column="0">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="6" column="1">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="6" column="2">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="6" column="3">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="6" column="4">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="6" column="5">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="6" column="6">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="6" column="7">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="6" column="8">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="7" column="0">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="7" column="1">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="7" column="2">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="7" column="3">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="7" column="4">
        <property name="text">
         <string/>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="7" column="5">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="7" column="6">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="7" column="7">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="7" column="8">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="8" column="0">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="8" column="1">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="8" column="2">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="8" column="3">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="8" column="4">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="8" column="5">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="8" column="6">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="8" column="7">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
       <item row="8" column="8">
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="solveButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Solve</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="effectsCheckBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Enable awesome sound effects</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="resultLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Ready</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
